### ISSUE TITLE
Mapping NAICS to different time periods

### ISSUE THREAD

**Issue by dwcaraway** [18F/afsmallbiz#49]
A user may submit a naics code in one year (e.g. 2012) and request an equivalent set of near-matches for another year (e.g. 2007).

Systems like FPDS have procurement information in 2007 and ~2000 NAICS so to reference historical information, we need to know what NAICs to look at based on current NAICS.

**Comment by catherinedevlin** [comment]
http://www.census.gov/cgi-bin/sssd/naics/naicsrch?chart=2002 has tables of 2012-to-2007 and 2007-to-2002 NAICS mappings - that's not what you mean by "near-matches", is it?  Otherwise not sure how we'll determine near-matchiness...

**Comment by dwcaraway** [comment]
Yup, the 2012-to-2007 and 2007-to-2002 are what I was referring to.
